docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -119,7 +126,7 @@ frame.add({ time: 1589189388597, content: 'user registered', level: 'info' });
 frame.add({ time: 1589189406480, content: 'unknown error', level: 'error' });
 ```
 
-### Unique log lines
+#### 'id' for assigning unique identifiers to log lines
```

Comment:
Is there a more descriptive heading we could use here? For example, if the goal is to improve support for dedupe by adding an 'id' field, should that information be incorporated into the heading? Or even "Use `id` to support log dedupe". What do you think is the main goal that this section is wanting to help the user accomplish?